### PR TITLE
Feature/multischeduler

### PR DESCRIPTION
**Pull request by PestusAtSAG**

### CODE REVIEW

File: basic/pom.xml [hunk 1]
```diff
@@ -66,7 +66,7 @@
 				<artifactId>spring-boot-maven-plugin</artifactId>
 				<version>${spring-boot-dependencies.version}</version>
 				<configuration>
-					<mainClass>com.c8y.ms.templates.basic.App</mainClass>
```
Comment (CGuether) [inline basic/pom.xml:69]: wrong file. This is the pom for the basic microservice which you have updated

File: basic/src/main/java/com/c8y/ms/templates/multithreading/App.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package com.c8y.ms.templates.basic;
+package com.c8y.ms.templates.multithreading;
```
Comment (CGuether) [inline basic/src/main/java/com/c8y/ms/templates/multithreading/App.java:1]: renamed package for wrong project

File: basic/src/main/java/com/c8y/ms/templates/multithreading/controller/DeviceController.java [hunk 3]
```diff
@@ -1,7 +1,7 @@
-package com.c8y.ms.templates.basic.controller;
```
Comment (CGuether) [inline basic/src/main/java/com/c8y/ms/templates/multithreading/controller/DeviceController.java:1]: renamed package for wrong project

File: basic/src/main/java/com/c8y/ms/templates/multithreading/controller/MeasurementController.java [hunk 4]
```diff
@@ -1,6 +1,6 @@
-package com.c8y.ms.templates.basic.controller;
```
Comment (CGuether) [inline basic/src/main/java/com/c8y/ms/templates/multithreading/controller/MeasurementController.java:1]: renamed package for wrong project

**Review by CGuether — CHANGES_REQUESTED** [review]
Revert your changes for basic project and rename the packages back to basic

File: basic/pom.xml (hunk 1 above)
Comment (PestusAtSAG) [reply basic/pom.xml]: i restored complete basic module from main branch

**Review by CGuether — APPROVED** [review]
can be merged

**Pull request merged**